### DISCUSSION TITLE
Is it possible to host multiple projects under the same domain?

### DISCUSSION THREAD

**Discussion by ninest** [vercel/vercel#4473] [Help] [answered]
I have a project which is hosted on `appname.now.sh`. I have two projects, one for an API and one for it's documentation, so I was wondering if it's possible to have something like this:

- `appname.now.sh/server` -> points to API built with Python/FastAPI
- `appname.now.sh/docs` -> points to the documentation static site

The server and docs are two separate projects.

One solution is to use Now's [serverless functions](https://vercel.com/docs/v2/serverless-functions/supported-languages#python). This will work, but I'd like to use FastAPI or Flask.

I am unsure if I can link both projects using [project linking](https://vercel.com/docs/cli#commands/overview/project-linking). I tried it, but I wasn't sure what exactly it is supposed it do.

Is this possible with Vercel?

**Comment by styfle** [comment, marked as answer]
Are the projects in a single repo? I would suggest using serverless functions in the `/api` folder as you mentioned. These will work with flask. Your frontend can use a static site generator like Next.js or Hugo and emit a `public` directory with your docs.

If these are two repos, they will need to be two separate Projects.

If you wish, you could create a 3rd project that sits in front of the two projects and assigns the domain and configures a couple rewrites in `vercel.json`: https://vercel.com/docs/configuration#project/rewrites
